### PR TITLE
Including html forms in markdown

### PR DESCRIPTION
**Pull request by jalagari**
Please ensure your pull request adheres to the following guidelines:
- [ ] make sure to link the related issues in this description
- [ ] when merging / squashing, make sure the fixed issue references are visible in the commits, for easy compilation of release notes

## Related Issues #379 

Thanks for contributing!

### CODE REVIEW

**Review by tripodsan — CHANGES_REQUESTED** [review]
sorry, but this is cannot be supported as it doesn't conform the markup semantics of the helix pipeline.

**Comment by jalagari** [comment]
Discussion at https://cq-dev.slack.com/archives/C05QU7MMRNF/p1705690163398589

File: src/process-forms.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+            children: [{
+              type: 'element',
+              tagName: 'td',
+              children: [{
+                type: 'element',
+                tagName: 'pre',
+                children: [{
+                  type: 'text',
+                  tagName: 'code',
+                  value: JSON.stringify(formDef),
```
Comment (trieloff) [inline src/process-forms.js:49]: We should not dump raw JSON into the content bus.

**Review by trieloff — CHANGES_REQUESTED** [review]
Let's find a way that is architecturally consistent with the overall Helix ecosystem.

**Pull request closed without merging**